codelists/statistic.csv

```diff
@@ -9,7 +9,11 @@ bidders,qualifiedBidders,Qualified bidders,The total number of unique organizati
 bidders,disqualifiedBidders,Disqualified bidders,The total number of unique organizations or consortia that did not pass the qualification stage of the evaluation process.
 EU,electronicBids,Electronic bids,The number of bids received by electronic means.
 EU,smeBids,Bids from SMEs,The number of bids received from small and medium-sized enterprises.
+EU,microBids,Bids from micro companies,The number of bids received from micro-sized enterprises.
+EU,smallBids,Bids from small companies,The number of bids received from small-sized enterprises.
+EU,mediumBids,Bids from medium companies,The number of bids received from medium-sized enterprises.
 EU,foreignBids,Bids from foreign firms,The number of bids received from bidders from outside the country where the tender is issued.
 EU,foreignBidsFromEU,Bids from firms in other European Economic Area countries,The number of bids received from tenderers from other European Economic Area countries.
 EU,foreignBidsFromNonEU,Bids from firms in non-European Economic Area countries,The number of bids received from tenderers from non-European Economic Area countries.
 EU,tendersAbnormallyLow,Tenders excluded because they were abnormally low,"The number of tenders excluded because they were abnormally low. Note that in some EU datasets this might have been converted from a boolean, such that a value of 1 would indicate ""1 or more"" tenders were excluded. Users need to evaluate and interpret data accordingly."
+EU,tendersDisqualified,Excluded tenders,"The number of tenders that were found inadmissible. A tender is found inadmissible where it has been verified that it has been submitted by a tenderer who has been excluded or who did not meet the selection criteria, or when it is not in conformity with the technical specifications, or is irregular (e.g. it was received late, having an abnormally low price or cost), unacceptable or unsuitable."
```

Comment:
This now matches the bidStatus codelist. I'll leave the description as-is, since it's in some ways better than the code's description in the bidStatus codelist, but we can look at those in a new issue.

```suggestion
EU,disqualifiedBids,Disqualified bids,"The number of bids that were found inadmissible. A tender is found inadmissible where it has been verified that it has been submitted by a tenderer who has been excluded or who did not meet the selection criteria, or when it is not in conformity with the technical specifications, or is irregular (e.g. it was received late, having an abnormally low price or cost), unacceptable or unsuitable."
```